### PR TITLE
Feature/tailwind loading fonts

### PR DESCRIPTION
**Pull request by illiecemessaoudi**
Using the same font than play SRG, 

Default font for all the project

### CODE REVIEW

File: app/styles/SRGSSRType.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@font-face {
+  font-family: 'SRGSSRType';
+  src:
+    url('/fonts/SRGSSRType.woff2') format('woff2'),
+    url('/fonts/SRGSSRType.woff') format('woff');
```
Comment (wetteyve) [inline app/styles/SRGSSRType.css:5]: in playSRG there was only '`woff2`' and no '`woff`' as far as i remember? have you added it or am i just not looking for it properly enough? :smile:

**Review by wetteyve — APPROVED** [review]
in playSRG there als were [overrides for two fallback fonts](https://github.com/SRGSSR/play-web/blob/4c2251df0cd11ecb3c097f287bb8d89173f5fa5d/packages/play-core/styles/GlobalStyles.tsx#L138). i think woe should adopt these as well, but thats something not really necessary right now, we just shouldnt forget about later on.  looks nice :+1:

File: app/styles/SRGSSRType.css (hunk 1 above)
Comment (illiecemessaoudi) [reply app/styles/SRGSSRType.css:5]: i was font-facing it with https://transfonter.org/  who give me the 2 format, it seems to be useful on some specific browser 

https://caniuse.com/?search=woff

Comment (wetteyve) [reply app/styles/SRGSSRType.css:5]: very nice, something new learned today :+1:

**Review by GauthierBosson — APPROVED** [review]

**Pull request merged**